src/stylesheets/components/_section.scss

```diff
@@ -1,37 +1,37 @@
 .usa-section {
-  @include u-padding-y($site-margins);
+  @include u-padding-y(units(4));
 
-  @include media($medium-screen) {
-    @include u-padding-y(6rem);
+  @include at-media('tablet') {
```

Comment:
Should be mobile large (mobile-lg)

Reply:
It looks like `medium-screen` is defined at `600px`, so the closest analog to that seems to be `tablet`, at `640px`. mobile-lg is 480; that seems like a pretty significant difference.

Reply:
I'm inclined to choose the closer value unless there's a better reason to go down to `mobile-lg`